pheromone.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import colors
+import matplotlib as mpl
+from point import Point
+class pheromone(object):
```

Comment:
Consider designing pheromone along the lines of a mock object, because we intend to use it for potential based path finding.